lerna.json

```diff
@@ -2,7 +2,7 @@
   "lerna": "3.16.4",
   "command": {
     "run": {
-      "scope": ["msal", "@azure/msal-common", "@azure/msal-browser"]
+      "scope": ["msal", "@azure/msal-common", "@azure/msal-browser", "@azure/msal-node"]
```

Comment:
Did this bootstrap the packages correctly? Please test and make sure.

Reply:
`msal-browser` will not compile; but `msal-common` is compilable and tests pass. Do you want me to make another PR reflecting `msal-browser`? I can once we know this branch is merge ready.

Reply:
yeah it isn't good that one package isn't compiling, we need to make sure everything compiles before we merge this (at least to dev).

Reply:
It is intentional to not touch msal-browser in the same PR. I plan to follow up and group the PRs before merge and ensure linkage. The PRs are kept simple for review ease. I already am making the needed changes.

Reply:
Just FYI lerna is now running in our CI, so adding this will run node as part of CI build and test. Make sure this is something you want before committing.